e2e/container/run_test.go

```diff
@@ -50,6 +50,82 @@ func TestRunWithContentTrust(t *testing.T) {
 	})
 }
 
+func TestUntrustedRun(t *testing.T) {
+	dir := fixtures.SetupConfigFile(t)
+	defer dir.Remove()
+	image := "registry:5000/alpine:untrusted"
```

Comment:
nit: we should probably use the `registryPrefix` const here; https://github.com/docker/cli/blob/2c4de4fb5e6b558fb775f515bac25d8ee83191ee/e2e/container/run_test.go#L16